### PR TITLE
Fix: References

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] fixes references to otherwise unresolvable Markdown files

🤷 To be honest, it's a bit disappointing that the documentation is in such a bad shape. We are paying customer of Spryker, and when we ask support, they often point to the documentation.

### CODE REVIEW

File: docs/02-development/01-choosing-a-mount-mode.md [hunk 1]
```diff
@@ -75,8 +75,6 @@ To configure a mount mode, see [Configuring a mount mode](07-configuring-a-mount
 
 Synchronization mount modes, such as mutagen or docker-sync, use algorithms to synchronize your code between host machine and a docker volume. This allows you to run applications at full speed avoiding file system mount latency.
 
-![](../images/mutagen-diagram.png)
```
Comment (localheinz) [inline docs/02-development/01-choosing-a-mount-mode.md:78]: Image does not exist.

**Comment by alexanderM91** [comment]
Hi @localheinz,
thanks a lot for your contribution. 

@andriitserkovnyi, could you please take a look on it? 
Thank you.

**Comment by localheinz** [comment]
Thank you, @alexanderM91 and @andriitserkovnyi!

**Comment by alexanderM91** [comment]
> Thank you, @alexanderM91 and @andriitserkovnyi!

Hey @localheinz,
again, thanks a lot for your contribution. 
Together we will make it much more better :muscle:

**Pull request merged**